packages/@coorpacademy-components/src/organism/setup-header/style.css

```diff
@@ -14,7 +14,7 @@
   width: 100%;
   height: 80px;
   box-sizing: border-box;
-  z-index: 800;
+  z-index: 1100;
```

Comment:
why do you change the z-index?

Reply:
'Cause the dot in lookandfeel component is in front of header component
![screenshot from 2018-11-27 10-35-32](https://user-images.githubusercontent.com/45251186/49057088-b98a4a00-f230-11e8-936c-d87a361638f9.png)